api/src/main/resources/CES_Drug_List-1.csv

```diff
@@ -0,0 +1,128 @@
+uuid,openboxes,name_es,concept,packaging
+85bd727f-08f9-41a4-b54a-9b8529b89027,,ACEITE LUBRICANTE PARA PIEZA DE MANO DE BAJA VELOCIDAD - ENVASE C/240 ML.,CIEL:162138,
+69bf13c3-6204-487a-86f4-ac07bfe8caa8,,AMLODIPINO TABLETAS O CAPSULAS 5 MG. - ENVASE CON 10 Ó 30 CAPSULAS O TABLETAS,CIEL:71138,
+c21a29c8-f44c-46a2-a268-e1248ff4ec56,,"AMLODIPINO, BESILATO DE DE 5 MG, TABLETA O CÁPSULA, ENVASE CON 10 TABLETAS O CÁPSULAS. - 10 TABLETAS O CÁPSULAS.",CIEL:71138,
+b9e113bb-cff7-4b7f-add7-9a4c83d1bb44,,"BENZOILO, PERÓXIDO DE DE 5 G / 100 ML, LOCIÓN DÉRMICA, ENVASE CON 30 ML. - ENVASE CON 30 ML",CIEL:72063,
+62866332-6e8a-4c8e-8779-3d32762b75c2,,"BENZOILO, PERÓXIDO DE DE 5 G / 100 ML, LOCIÓN DÉRMICA, ENVASE CON 50 ML. - ENVASE CON 50 ML",CIEL:72063,
+341dbcae-5bb1-4798-b20b-47ac8a755dcb,,"BENZOILO, PERÓXIDO DE DE 5 G / 100 G, GEL DÉRMICO, ENVASE CON 60 G. - ENVASE CON 60 ML",CIEL:72063,
+ad1c5007-57a7-4bb7-afd4-e450d307486b,,"CEFALEXINA MONOHIDRATADA DE 500 MG, TABLETA O CÁPSULA, ENVASE CON 20 TABLETAS O CÁPSULAS. - ENVASE CON 20 TABLETAS O CÁPSULAS.",CIEL:73087,
+42e907a0-8b22-4b54-9135-2551525c2f8a,,"COMPLEJO B DE TIAMINA, MONONITRATO DE, 100MG, PIRIDOXINA 5 MG, CIANOCOBALAMINA 50 µG, TABLETA, COMPRIMIDO O CÁPSULA, ENVASE CON 30 TABLETAS, COMPRIMIDOS O CÁPSULAS. - 30 TABLETAS, ",CIEL:84995,
```

Comment:
Truncating at 180 characters seemed to fix the max length problem. I wasn't able to actually locate the maximum length setting anywhere. The hibernate mapping and database both say they'll accept something up to 255 characters, but the error was being generated on a string of length 190 or so.